test/tictactoe/game_runner_test.clj

```diff
@@ -1,73 +1,88 @@
 (ns tictactoe.game-runner-test
   (:require [clojure.test :refer [deftest is testing]]
-            [tictactoe.game-runner :as game-runner]))
+            [tictactoe.game-runner :refer [start sleep-time]]
+            [clojure.string :as string]))
 
 (defn make-input
   [coll]
   (apply str (interleave coll (repeat "\n"))))
 
-(defn lines
-  [& args]
-  (apply str (interpose "\n" args)))
+(deftest test-game-play-human-v-human
+  (testing "Player one wins the game on a 3x3 board via row"
+  (is (= true (string/includes?
+         (with-out-str ""
+           (with-in-str (make-input '("1" "1" "X" "O" "0" "5" "1" "4" "2")) 
+            (binding [sleep-time 0]
```

Comment:
indentation

Reply:
I will fix these - I formatted them using vim and was trying to keep things consistent.

Reply:
👍